staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -113,17 +119,16 @@ func BuildAndRegisterAggregator(downloader *Downloader, delegationTarget server.
 
 	// Build initial spec to serve.
 	klog.V(2).Infof("Building initial OpenAPI spec")
-	defer func(start time.Time) {
-		duration := time.Since(start)
-		klog.V(2).Infof("Finished initial OpenAPI spec generation after %v", duration)
-
-		regenerationCounter.With(map[string]string{"apiservice": "*", "reason": "startup"})
-		regenerationDurationGauge.With(map[string]string{"reason": "startup"}).Set(duration.Seconds())
-	}(time.Now())
+	start := time.Now()
 	specToServe, err := s.buildOpenAPISpec()
 	if err != nil {
 		return nil, err
 	}
+	duration := time.Since(start)
```

Comment:
Are we planning on having a similar counter for v3?

Reply:
This metric is for the duration for aggregation, we avoid aggregation in OpenAPI v3 so I'm not sure if it'll be useful at all?